providers/fips/fipsprov.c

```diff
@@ -223,14 +239,237 @@ static int dummy_evp_call(void *provctx)
+        NULL
+    };
+    static const char *dh_g_hex[] = {
+        "5ef7b88f2df60139351dfbfe1266805fdf356cdfd13a4da0050c7ede"
+        "246df59f6abf96ade5f2b28ffe88d6bce7f7894a3d535fc82126ddd4"
+        "24872e16b838df8c51e9016f889c7c203e98a8b631f9c72563d38a49"
+        "589a0753d358e783318cefd9677c7b2dbb77d6dce2a1963795ca64b9",
+        "2d1c9aac6d0e8d431de5e50060dff78689c9eca1c1248c16ed09c7ad",
+        "412a17406d2b525aa1cabb237b9734ec7b8ce3fae02f29c5efed30d6"
+        "9187da109c2c9fe2aadbb0c22af54c616655000c431c6b4a379763b0"
+        "a91658efc84e8b06358c8b4f213710fd10172cf39b830c2dd84a0c8a"
+        "b82516ecab995fa4215e023e4ecf8074c39d6c88b70d1ee4e96fdc20",
+        "ea115c32",
+        NULL
     };
-    unsigned int dgstlen = 0;
+    static const char *dh_priv_hex[] = {
+        "1433e0b5a917b60a3023f2f8aa2c2d70d2968aba9aeac81540b8fce6",
+        NULL
+    };
+    static const char *dh_pub_hex[] = {
+        "95dd338d29e5710492b918317b72a36936e1951a2ee5a5591699c048"
+        "6d0d4f9bdd6d5a3f6b98890c62b37652d36e712111e68a7355372506"
+        "99efe330537391fbc2c548bc5ac3e5b23386c3eef5eb43c099d70a52"
+        "02687e83964248fca91f40908e8fb3319315f6d2606d7f7cd52cc6e7",
+        "c5843afb22519cf0f0f9d3a0a4e8c88899efede7364351fb6a363ee7"
+        "17e5445adab4c931a6483997b87dad83677e4d1d3a7775e0f6d00fdf"
+        "73c7ad801e665a0e5a796d0a0380a19fa182efc8a04f5e4db90d1a86"
+        "37f95db16436bdc8f3fc096c4ff7f234be8fef479ac4b0dc4b77263e",
+        "07d9959de0f1bf3f0ae3d9d50e4b89c99e3ea1217343dd8c6581acc4"
+        "959c91d3",
+        NULL
+    };
+    static const char *dh_peer_pub_hex[] = {
+        "1fc1da341d1a846a96b7be24340f877dd010aa0356d5ad58aae9c7b0"
+        "8f749a32235110b5d88eb5dbfa978d27ecc530f02d3114005b64b1c0"
+        "e024cb8ae21698bca9e60d42808622f181c56e1de7a96e6efee9d665"
+        "67e91b977042c7e3d0448f05fb77f522b9bfc8d33cc3c31ed3b31f0f",
+        "ecb6db4f6ea311e77afdbcd47aee1bb150f216873578fb96468e8f9f"
+        "3de8efbfce75624b1df05322a34f1463e839e8984c4ad0a96e1ac842"
+        "e5318cc23c062a8ca171b8d575980dde7fc56f1536523820d43192bf"
+        "d51e8e228978aca5b94472f339caeb9931b42be301268bc99789c9b2",
+        "5571c3c0e4cb3f007f1a511cbb53c8519cdd1302abca6c0f34f96739"
+        "f17ff48b",
+        NULL
+    };
+    static const char *dh_secret_exptd_hex[] = {
+        "08ff33bb2ecff49a7d4a7912aeb1bb6ab511641b4a76770c8cc1bcc2"
+        "33343dfe700d11813d2c9ed23b211ca9e8786921edca283c68b16153"
+        "fa01e91ab82c90ddab4a95816770a98710e14c92ab83b6e46e1e426e"
+        "e852430d6187daa3720a6bcd73235c6b0f941f3364f50420551a4bfe",
+        "afe2bc438505a59a4a40daca7a895a73db575c74c13a23ad8832957d"
+        "582d38f0a6165fb0d7e9b8799e42fd3220e332e98185a0c9429757b2"
+        "d0d02c17dbaa1ff6ed93d7e73e241eaed90caf394d2bc6570f18c81f"
+        "2be5d01a2ca99ff142b5d963f9f500325e7556f95849b3ffc7479486",
+        "be1d4596a3106bd5cb4f61c57ec5f100fb7a0c82a10b82526a97d1d9",
```

Comment:
Drop the comma here?  It's four lines concatenating previously but one here which isn't consistent.

Reply:
There's a limit on static stringth length, so perhaps put a comma on all lines and glue them together as is done elsewhere?